docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,52 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main feature of the Metadata API is to extract metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
+
```

Comment:
As the line 3 and line 5 are talking about the same content, how about removing line 3?

Also as we changed the second sentence into the overview sentence of this page,
it is better to give only simple description from overview while offering detailed description to the followed section.

So how about changing the sentence, for example, like below?

Metadata contains various information of multimedia files. You can [extract metadata](#extract-metadata) from multimedia files.

Reply:
@mijin85cho current functionality is a 'basic' part, which is planned to be extended in future. Used 'template' is consistent with other modules which uses one sentence of module summary, then the list of more detailed descriptions of features.

Planned features will  include: getting artwork, frames and synclyrics for video.
I agree that current description looks too simple, but for now the module is just as simple. We can use the form you proposed, which will be changed when the list of supported features will be extended, is it ok?

Reply:
@mijin85cho 

Your suggestion is not contextually aligned (contains word slightly changes it)
How about this : 
Metadata is the information about multimedia files. You can extract metadata from multimedia files.

Reply:
@pkosko I guess the planned features are also included in Tizen 6.0. 
As you said, I suggested sentence modification because of simple module. If you're planning to add additional features, for now let's keep the current.

Reply:
> Your suggestion is not contextually aligned (contains word slightly changes it)
> How about this :
> Metadata is the information about multimedia files. You can extract metadata from multimedia files.

I found your comment after I replied to @pkosko. Sorry for the miss.

@pkosko One more suggestion. How about adding @nawab-samsung's suggested sentence to the overview?

Metadata is the information about multimedia files.

Reply:
@pkosko can you please incorporate this comment, following which I will do a full review and approve if no more changes are required.

Reply:
It is done in line 3.

Reply:
@pkosko  thanks a lot